src/methods/method_overloading/medium/q7/OverloadDubDub.java

```diff
@@ -0,0 +1,16 @@
+package methods.method_overloading.medium.q7;
+
+/*
+Overload a Java Method so that it can take either a double variable, or a double variable and an integer variable. The
```

Comment:
```suggestion
Overload a Java Method called multiply so that it can take either a double variable or a double variable and an integer variable. The
```